src/lib/support/DefaultStorageKeyAllocator.h

```diff
@@ -299,17 +299,15 @@ class DefaultStorageKeyAllocator
     }
 
     // Stores the client cert payload for the given fabric & endpoint
-    // (excepting intermediate cert, see TlsClientCertEntityIntermediateKey)
     static StorageKeyName TlsClientCertEntityKey(FabricIndex fabric, EndpointId endpoint, uint16_t idx)
     {
         return StorageKeyName::Formatted("f/%x/e/%x/tlsc/%x", fabric, endpoint, idx);
     }
 
-    // Stores an intermediate cert payload for a client cert for the given fabric & endpoint
-    static StorageKeyName TlsClientCertEntityIntermediateKey(FabricIndex fabric, EndpointId endpoint, uint16_t entity_idx,
-                                                             uint8_t intermediate_idx)
+    // Stores global data about certificates for the given endpoint, across all fabrics
```

Comment:
This should be documented as being used by a specific implementation, with probably a pointer to the interface to implement for applications that have no concept of "storage across all fabrics".

Reply:
Added doc